README.md

```diff
@@ -33,85 +33,11 @@
 
 ## What is Meshtastic?
 
-<table>
-<tr>
-<td>
 <p>
```

Comment:
Since we've removed much of the HTML from much of the document, could we just make this portion vanilla markdown now?

Reply:
Line 36+, don't know why that didn't save correctly.

Reply:
Good point, html removed